src/pydap/handlers/dap.py

```diff
@@ -46,12 +49,12 @@ def __init__(self, url):
         ddsurl = urlunsplit((scheme, netloc, path + '.dds', query, fragment))
         r = requests.get(ddsurl)
         r.raise_for_status()
-        dds = r.text.encode('utf-8')
+        dds = r.text.decode('ascii')
```

Comment:
Isn't this the result from `requests.get` (of _the_ requests module)? `r.text` should already be decoded.